src/components/ContactsDel.js

```diff
@@ -0,0 +1,19 @@
+import { useNavigate, useParams } from "react-router-dom";
+
+export default function ContactsDel (props) {
+  const navigate = useNavigate()
+  const params = useParams()
+  const { setContacts } = props
+  function deleteMe () {
```

Comment:
So when you deleted a contact you did not delete messages? I did the same but I guess this is not ideal. Did you think about using adding an extra key to the contact objects with an array of meetings?

Reply:
do u mean deleting the meeting of a deleted contact? Yes I did think of this and I good notice I will change this